steps/setup_docker_continue_codespace_generic.yml

```diff
@@ -0,0 +1,23 @@
+id: setup_docker_continue_codespace_generic
+hints:
+- Make sure your codespace has started and is ready to use.
+startFlow:
+trigger:
+  type: github_codespace_started
+  flowNode:
+    if:
+      conditions:
+      - conditionId: user_has_feature_flag
+        params:
+          featureFlag: 'codespaces'
```

Comment:
I think this is redundant, no? If he got to this trigger, he has the feature flag

Reply:
Me :(

Reply:
I answered similarly to shem on the engine PR, copying here:
> theoretically once we start the experiment we will have to open codespaces for all outside collaborators (in github) so a user could manually open a codespace and it will send us the webhook. i'd like to have control with these types of checks while this is behind a feature flag

Reply:
Fair enough